config/crd/kustomization.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - https://github.com/kong/kubernetes-configuration/config/crd/gateway-operator?ref=v2.0.0-alpha.5 # Version is auto-updated by the generating script.
+  - gateway-operator
```

Comment:
Should we use another name like `kong-operator` instead of `gateway-operator`?

Reply:
I only modified config/crd/gateway-operator to config/crd/kong-operator. I think it's not appropriate to make too many adjustments right now.